### DISCUSSION TITLE
Need help with migrating from v1

### DISCUSSION THREAD

**Discussion by FrancoTanzarella** [jpudysz/react-native-unistyles#84] [Q&A] [answered]
I am following the docs to migrate from v1 to v2 but I'm unsure of what to import in my `App.tsx` from `unistyles.ts`.  Sorry if this is obvious.

```
Don’t forget to import the unistyles.ts file somewhere in your code eg. in the App.tsx file.
```

This is my unistyles.ts file

```javascript
import { UnistylesRegistry } from 'react-native-unistyles';
import { lightTheme, darkTheme } from '../styles/themes';

type AppThemes = {
  light: typeof lightTheme;
  dark: typeof darkTheme;
};

declare module 'react-native-unistyles' {
  export interface UnistylesThemes extends AppThemes {}
}

UnistylesRegistry.addThemes({
  light: lightTheme,
  dark: darkTheme,
}).addConfig({
    initialTheme: 'light'
});
```

**Comment by jpudysz** [comment, marked as answer]
No worries 👋. This code needs to run somehow, right?
If you don't import anything metro will skip it. 

Open your App.tsx and add following import (I'm on my mobile phone so I can't format it):

import '../unistyles'

And that's it!

**Reply by FrancoTanzarella** [reply]
Obviously! I can overthink things sometimes too much. Thank you.

**Reply by ammarfaris** [reply]
To be honest, it delays me a bit also, would be great to include that in the [docs](https://reactnativeunistyles.vercel.app/start/setup/) and the box that says `Don’t forget to import the unistyles.ts file somewhere in your code eg. in the App.tsx file.` is also not highlighted as much, so I also missed that info initially, unfortunately. 

@jpudysz

**Reply by jpudysz** [reply]
Got it, guys! Thanks for pointing it out. I will bold it in the same manner as other things.
